Library/Dungeon Fantasy RPG/Dungeon Fantasy RPG Equipment Modifiers.eqm

```diff
@@ -0,0 +1,177 @@
+					"cost": "+47 CF",
+					"weight_type": "to_base_weight",
+					"weight": "x2",
+					"notes": "Heavy leather; additional +4 DR vs fire"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "53b65396-65a2-410e-82ae-25c69ecdd641",
+					"name": "Dwarven (+1 DR)",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+4 CF",
+					"notes": "Plate"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "8fac7fc7-3c6a-4c11-b1de-0586d56f1f04",
+					"name": "Fine",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+9 CF",
+					"weight_type": "to_base_weight",
+					"weight": "x3/4"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "d4f0ad21-22a9-4a81-8954-329c2e27bdda",
+					"name": "Giant-Spider Silk (+1 DR)",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+99 CF",
+					"weight_type": "to_base_weight",
+					"weight": "+0 lb",
+					"notes": "Light cloth"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "903fa020-4a41-4bdf-9785-664685343be7",
+					"name": "Meteoric",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+19 CF",
+					"notes": "Any metal"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "1a955050-1485-4a3e-98b9-1d28444c9b08",
+					"name": "Orichalcum",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+29 CF",
+					"weight_type": "to_base_weight",
+					"weight": "x1/3",
+					"notes": "Any plate"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "da7aab67-2adb-455c-8371-2ac173df1f2c",
+					"name": "Ornate +1",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+1 CF"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "442ae4d5-5356-4233-ad9b-74f0d975b324",
+					"based_on_id": "da7aab67-2adb-455c-8371-2ac173df1f2c",
+					"based_on_hash": "/2l0LUbDuSYWxuQW8EV/Wbd3yIsjmOpK8F4FbLQxjvQ",
+					"name": "Ornate +2",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+4 CF"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "4f4c08e9-09c1-42b0-8b98-aa6d4ee0ad5a",
+					"based_on_id": "442ae4d5-5356-4233-ad9b-74f0d975b324",
+					"based_on_hash": "QBRpW6RotdtEPi2TaXZ9zfLdS3BU5/hguue7lUBe0hM",
+					"name": "Ornate +3",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+9 CF"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "c98dc896-6a6c-445e-907e-d8431820c355",
+					"name": "Spiked",
+					"reference": "DFA110",
+					"cost_type": "to_base_cost",
+					"cost": "+2 CF",
+					"notes": "Any plate that covers body or more"
+				},
+				{
+					"type": "eqp_modifier",
+					"version": 1,
+					"id": "90d015a6-b379-4b63-ab4d-93b86e7f738d",
+					"name": "Thieves'",
```

Comment:
Similar to ornate, doesn't this grant some sort of skill modifier to stealth? Again, if so, that should be added as a feature to this entry.

Reply:
Added a note. It's pretty fiddly.